pom.xml

```diff
@@ -65,6 +65,13 @@
       <version>2.9.0</version>
     </dependency>
 
+    <!-- Datadog Tracer (only needed if you do not use dd-java-agent) -->
+    <dependency>
+      <groupId>com.datadoghq</groupId>
+      <artifactId>dd-trace-ot</artifactId>
+      <version>0.30.0</version>
```

Comment:
Any reason you're using `0.30.0`?  Our latest release is `0.44.0`.

Reply:
The max i can do is `0.32.0` then i start to have dependency conflicts

Reply:
```
[WARNING] Rule 4: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.slf4j:slf4j-api:1.7.25 paths to dependency are:
+-org.datadog.jenkins.plugins:datadog:1.0.3-SNAPSHOT
  +-com.datadoghq:dd-trace-ot:0.33.0
    +-org.slf4j:slf4j-api:1.7.25
and
+-org.datadog.jenkins.plugins:datadog:1.0.3-SNAPSHOT
  +-org.slf4j:log4j-over-slf4j:1.7.25
    +-org.slf4j:slf4j-api:1.7.25 (managed) <-- org.slf4j:slf4j-api:1.7.28
and
+-org.datadog.jenkins.plugins:datadog:1.0.3-SNAPSHOT
  +-org.slf4j:jcl-over-slf4j:1.7.25
    +-org.slf4j:slf4j-api:1.7.25 (managed) <-- org.slf4j:slf4j-api:1.7.28
and
+-org.datadog.jenkins.plugins:datadog:1.0.3-SNAPSHOT
  +-org.slf4j:slf4j-jdk14:1.7.25
    +-org.slf4j:slf4j-api:1.7.25 (managed) <-- org.slf4j:slf4j-api:1.7.28
and
+-org.datadog.jenkins.plugins:datadog:1.0.3-SNAPSHOT
  +-com.datadoghq:dd-trace-ot:0.33.0
    +-com.datadoghq:dd-trace-api:0.33.0
      +-org.slf4j:slf4j-api:1.7.25 (managed) <-- org.slf4j:slf4j-api:1.7.28
and
+-org.datadog.jenkins.plugins:datadog:1.0.3-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-war:2.138.4
    +-org.jenkins-ci.modules:sshd:2.4
      +-org.apache.sshd:sshd-core:1.7.0
        +-org.slf4j:slf4j-api:1.7.25 (managed) <-- org.slf4j:slf4j-api:1.7.28
```